automatic/_output/cdex/1.81/tools/chocolateyInstall.ps1

```diff
@@ -1,7 +1,8 @@
 $packageName = 'cdex'
 $installerType = 'exe'
 $silentArgs = '/S'
-$url = 'http://mirror.cdex.mu/CDex-1.81-win32.exe'
+#$url = 'http://mirror.cdex.mu/CDex-1.81-win32.exe'
+$url = 'https://download.heise.de/software/9b13eed3373cebf4aca3029b694fec64/57ddb0b5/191936/cdex-1.81-win32.exe'
```

Comment:
Indeed, it seems to be time-based. And I cannot find another credible source that offers this binary. Rather concerning :-/